apps/lidarr/Dockerfile

```diff
@@ -0,0 +1,62 @@
+ARG VERSION
+ FROM docker.io/library/alpine:3.21@sha256:a8560b36e8b8210634f77d9f7f9efd7ffa463e380b75e2e74aff4511df3ef88c
+
+ ARG TARGETPLATFORM
+ ARG VERSION
+
+ ENV \
+     COMPlus_EnableDiagnostics="0" \
+     LIDARR__UPDATE__BRANCH="develop"
+
+ ENV UMASK="0002" \
+     TZ="Etc/UTC"
```

Comment:
Move this above, like so: https://github.com/home-operations/containers/blob/main/apps/bazarr/Dockerfile#L7-L8